contrib/sort-txt.mjs

```diff
@@ -0,0 +1,78 @@
+import fs from "fs";
+import path from "path";
+
+if (process.argv.length <= 2) {
+  console.log(`Usage: node ./sort-txt.mjs path/to/cj3.txt`);
+}
+
+function block(ch) {
+  const cp = ch.codePointAt(0);
+  if (cp <= 0x4dbf && cp >= 0x3400) {
+    return 1; // Ext. A
+  } else if (cp <= 0xffff) {
+    return 0; // Basic + Misc
+  } else if (cp <= 0x2a6df) {
+    return 2;
+  } else if (cp <= 0x2b73f) {
+    return 3;
+  } else if (cp <= 0x2b81f) {
+    return 3; // Ext. C & Ext. D are mixed :(
```

Comment:
With the sorting utilities we can consider split Ext. C and Ext. D in `cj3.txt`, like we did on other extension blocks, even though the support of Ext. C and Ext. D may be introduced at the same time.